service/service.go

```diff
@@ -100,22 +100,25 @@ func New(config Config) (*Service, error) {
 		awsConfig = awsclient.Config{
 			AccessKeyID:     config.Viper.GetString(config.Flag.Service.AWS.AccessKey.ID),
 			AccessKeySecret: config.Viper.GetString(config.Flag.Service.AWS.AccessKey.Secret),
+			SessionToken:    config.Viper.GetString(config.Flag.Service.AWS.AccessKey.Session),
 		}
 	}
 
 	var awsHostConfig awsclient.Config
 	{
 		accessKeyID := config.Viper.GetString(config.Flag.Service.AWS.HostAccessKey.ID)
 		accessKeySecret := config.Viper.GetString(config.Flag.Service.AWS.HostAccessKey.Secret)
+		sessionToken := config.Viper.GetString(config.Flag.Service.AWS.HostAccessKey.Session)
 
-		if accessKeyID == "" && accessKeySecret == "" {
+		if accessKeyID == "" && accessKeySecret == "" && sessionToken == "" {
```

Comment:
Please remove the session token check here. When deployed the operator uses an IAM account that doesn't use MFA so it won't have a token.

Reply:
Done